python/pyspark/pandas/tests/data_type_ops/test_boolean_ops.py

```diff
@@ -572,10 +578,28 @@ def test_isnull(self):
     def test_astype(self):
         pser = self.pser
         psser = self.psser
+
+        # [True, False, <NA>] is returned in pandas
```

Comment:
I guess we should follow the pandas' behavior?
Could you file a JIRA ticket to track this and put the JIRA ID like `# TODO(SPARK-xxxxx): [True, ...`, or address here?

Reply:
Makes sense, SPARK-35976 is created and commented here.